Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -376,14 +376,13 @@ end CommSemiring
 
 section CommRing
 
-variable {R R₂ S S₂ M N P : Type*}
-variable {Mₗ Nₗ Pₗ : Type*}
-variable [CommRing R] [CommRing S] [CommRing R₂] [CommRing S₂]
+variable {R M : Type*}
+variable [CommRing R]
```

Comment:
optional: combine with the previous line?

Reply:
Also here.